nextstrain_profiles/100k/README.md

```diff
@@ -0,0 +1,21 @@
+## Aim
+
+To build a representative 100k dataset which is available for testing / developing builds locally.
+This is intended to run weekly via a GitHub action (which triggers a job to be run on AWS).
+It will make two files available:
+
+* `s3://nextstrain-ncov-private/100k/metadata.tsv.xz`
+* `s3://nextstrain-ncov-privatr/100k/sequences.fasta.xz`
+
+While this profile is not recommended to be run locally, you can see what rules would be run via:
+
+```
+snakemake --cores 1 --configfile nextstrain_profiles/100k/config.yaml -npf upload --dag | dot -Tpdf > dag.pdf
```

Comment:
```suggestion
snakemake -c1 --configfile nextstrain_profiles/100k/config.yaml -npf upload --dag | dot -Tpdf > dag.pdf
```

Can shorten ;)

Reply:
Going to leave as long form - hopefully we won't need to actually run this by hand very frequently and it's clearer (to me, at least)